docs/readme-cmake.md

```diff
@@ -151,15 +144,12 @@ If you want to use the Code::Blocks IDE, from the root of the project folder exe
 
 With `<preset>` being one of the available presets (see above for a list). For Windows for example you'd want `mingw`. Now there will be a Code::Blocks project inside `build\mingw`.
 
-
-
-## Building with XCode
+### Building with XCode
```

Comment:
Is it not possible to build with clang in CodeBlocks on Windows?
If mingw is the only option, clarifying near the beginning of the section that this can only be done with those build tools on Windows is important, I think.

Reply:
It is possible to use clang with CodeBlocks :)